vela-templates/internal/cue/webservice.cue

```diff
@@ -117,12 +116,16 @@ parameter: {
 	// +short=i
 	image: string
 
-	// +usage=Commands to run in the container
-	cmd?: [...string]
-
 	// +usage=Which port do you want customer traffic sent to
 	// +short=p
 	port: *80 | int
+
+	// If addRevisionLabel is true, the appRevision label will be added to the underlying pods
+	addRevisionLabel: *false | bool
```

Comment:
we should add some `keyword` such as `//+ignore` to let the user can't see the parameter and only be used by system.